src/main/interface/settings.c

```diff
@@ -447,7 +447,8 @@ const clivalue_t valueTable[] = {
     { "gyro_lma_depth",             VAR_UINT8  | MASTER_VALUE, .config.minmax = {0, 11}, PG_GYRO_CONFIG, offsetof(gyroConfig_t, gyro_lma_depth)},
     { "gyro_lma_weight",            VAR_UINT8  | MASTER_VALUE, .config.minmax = {0, 100}, PG_GYRO_CONFIG, offsetof(gyroConfig_t, gyro_lma_weight)},
 
-    { "moron_threshold",            VAR_UINT8  | MASTER_VALUE, .config.minmax = { 0,  200 }, PG_GYRO_CONFIG, offsetof(gyroConfig_t, gyroMovementCalibrationThreshold) },
+    { "gyro_calib_duration",        VAR_UINT16 | MASTER_VALUE, .config.minmax = { 50,  3000 }, PG_GYRO_CONFIG, offsetof(gyroConfig_t, gyroCalibrationDuration) },
+    { "gyro_calib_noise_limit",     VAR_UINT8  | MASTER_VALUE, .config.minmax = { 0,  200 }, PG_GYRO_CONFIG, offsetof(gyroConfig_t, gyroMovementCalibrationThreshold) },
```

Comment:
Maybe normalize it to DPS or make it a percentage of max DPS, hence 0-100%?

Reply:
I'm not sure we want to change both the name and the value at this time - could be a support nightmare with a lot of broken configs.  Or maybe we do and just rip the bandage off at one time.

I'm not sure about the scale though.  DPS doesn't seem usable to me.  The limit is compared to the standard deviation of the raw gyro samples and the default value of 48 only corresponds to 2.93 dps (for a 2000dps scale gyro).

If instead we normalized to a 100% scale this would make the new default value be 24.  But again it's just an arbitrary number that's half of what it used to be.  Also doing this makes it difficult to extend the upper boundary if we ever need to - maybe for the next generation of ultra noisy gyros. :(

Reply:
Yeah, I think the current way it is defined as having an open upper boundary is worth keeping (even if we impose an upper limit for what the parameter can be set to). So using a percent value is not a good way to go.